test/topology_experimental_raft/test_alternator.py

```diff
@@ -297,13 +297,24 @@ async def check_two_live_nodes():
     response = requests.get(localnodes_request)
     j = json.loads(response.content.decode('utf-8'))
     assert len(j) == 1
-    # Ending the test here will kill both servers. We don't wait for the
-    # second server to finish its long injection-caused bootstrap delay,
-    # so we don't check here that when the second server finally comes up,
-    # both nodes will finally be visible in /localnodes. This case is checked
-    # in other tests, where bootstrap finishes normally - we don't need to
-    # check this case again here.
-    task.cancel()
+
+    # We don't want to wait for the second server to finish its long
+    # injection-caused bootstrap delay, so we won't check here that when the
+    # second server finally comes up, both nodes will finally be visible in
+    # /localnodes. This case is checked in other tests, where bootstrap
+    # finishes normally, so we don't need to check this case again here.
+    # But we can't just finish here with "task" unwaited or we'll get a
+    # warning about an unwaited coroutine, and the ScyllaClusterManager's
+    # tasks_history will wait for it anyway. For the same reason we can't
+    # task.cancel() (this will cause ScyllaClusterManager's tasks_history
+    # to report the ScyllaClusterManager got BROKEN and fail the next test).
+    # Sadly even abruptly killing the servers (with manager.server_stop())
+    # (with the intention to then "await task" quickly) doesn't work,
+    # probably because of a bug in the library. So we "await task"
+    # anyway, and this test takes 2 minutes :-(
+    #for server in await manager.all_servers():
+    #    await manager.server_stop(server.server_id)
+    await task
 
 # TODO: add a more thorough test for /localnodes, creating a cluster with
 # multiple nodes in multiple data centers, and check that we can get a list
```

Comment:
Does the patch belong in the series? We might want to backport it if the test flakes a lot.

Reply:
Yes, I probably should. The code as it was wasn't flaky - if the broken test was the last test of the file it always passed. But if in the future we'll ever want to backport any other test to the same file after that test, we'll need to backport this specific change. So I'll split it to a separate PR.

Reply:
Unfortunately, our slow review and CI process prevents me from moving this patch into a separate series. If I do, it means it will take at least a day until this one patch goes in, and only then I can send a new version of this PR. And tomorrow (Iran willing) I'm going on vacation and will not be able to do it. So I'm leaving this patch in this series. 
It's not impossible to backport just a single patch from the whole series. What's harder is to remember to do it.